README.md

```diff
@@ -1,11 +1,14 @@
-# TakeShape Starter Auth0 + Stripe
+# TakeShape API Indexing Starter: Auth0 + Stripe
```

Comment:
```suggestion
# TakeShape Starter: Auth0 + Stripe
```

API indexing is a feature, prominently mention it in the text below but it doesn't need to be in the title.